dymaptic.Pro.ZoomToCoordinates/dymaptic.Pro.ZoomToCoordinates.csproj

```diff
@@ -9,12 +9,18 @@
     <NoWarn>CA1416</NoWarn>
     <Nullable>enable</Nullable>
   </PropertyGroup>
+  <ItemGroup>
+    <None Remove="Images\dymaptic-logo---circle.png" />
+    <None Remove="Images\dymaptic-logo-2048x585.png" />
+  </ItemGroup>
   <ItemGroup>
     <Content Include="Config.daml" />
-    <Content Include="Images\AddInDesktop16.png" />
-    <Content Include="Images\AddInDesktop32.png" />
-    <Content Include="DarkImages\AddInDesktop16.png" />
-    <Content Include="DarkImages\AddInDesktop32.png" />
+    <Resource Include="Images\dymaptic-logo---circle.png">
```

Comment:
I'm surprised that `CopyToOutputDirectory = Never` works with package building? Make sure the final version has your images in it.

Reply:
Good point, I'll definitely monitor.  The default images for an ArcGIS Pro AddIn are set to build action `Content` and `Copy to Output Directory = Do not copy` and come through - though they don't have the `CopyToOutputDirectory = Never` in the `.csproj` file.  I've deleted the AddIn a bunch of times and recompiled it anew, and the images are still coming across, but I'll make sure this occurs on the final digitally signed one.